packages/demo-react/package.json

```diff
@@ -11,8 +11,8 @@
   ],
   "dependencies": {
     "@deloitte-digital-au/webpack-config-react": "^0.0.1",
-    "react": "^16.3.1",
-    "react-dom": "^16.3.1"
+    "react": "16.3.2",
+    "react-dom": "16.3.2"
```

Comment:
I am going to unpin the dependencies in the demo packages, because these demo packages could possibly be used as starting points for new projects, and then we would want to use semver ranges for dependencies.